tests/Capgemini.PowerApps.PackageDeployerTemplate.IntegrationTests/PackageTemplateBaseTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿namespace Capgemini.PowerApps.PackageDeployerTemplate.IntegrationTests
+{
+    using System;
+    using System.Linq;
+    using FluentAssertions;
+    using Microsoft.Xrm.Sdk;
+    using Microsoft.Xrm.Sdk.Query;
+    using Xunit;
+
+    public class PackageTemplateBaseTests : IClassFixture<PackageDeployerFixture>
```

Comment:
I was trying to do them all but inevitably missed a few as there were so many 😄 I'll try to find the stragglers. I think I left some out in the test project as I didn't want to introduce constants if they were only used in the tests rather than the template itself, but I suppose it really doesn't matter if we have a few unused in the template.